etc/kayobe/stackhpc-overcloud-dib.yml

```diff
@@ -55,8 +55,10 @@ stackhpc_overcloud_dib_env_vars:
   # Avoid DNS queries during sudo commands, since we might not always have working DNS.
   DIB_SUDOERS_CONFIG: |
     Defaults  !fqdn
-  # FIXME: Support templating repo files.
-  # DIB_YUM_MINIMAL_BOOTSTRAP_REPOS: /path/to/dir/containing/dib-mirror-*.repo
+  # On Rocky Linux 9, use the host's repo files.
+  # These will have been configured to point to Test Pulp repo snapshots during the 'host configure' step.
+  # FIXME: This assumes we are building on RL9, which may not be the case.
+  DIB_YUM_MINIMAL_BOOTSTRAP_REPOS: "{% if os_distribution == 'rocky' and os_release == '9' %}/etc/yum.repos.d/{% endif %}"
```

Comment:
Unfortunately this didn't work:

```
cat /etc/rocky-release
Rocky Linux release 9.3 (Blue Onyx)
```

Reply:
Because DIB_YUM_MINIMAL_BOOTSTRAP_REPOS works for yum-minimal element, but rocky-container depends on yum element - working on it.